Emby.Server.Implementations/LiveTv/Listings/SchedulesDirect.cs

```diff
@@ -152,22 +155,22 @@ public async Task<IEnumerable<ProgramInfo>> GetProgramsAsync(ListingsProviderInf
                                 var imagesWithText = allImages.Where(i => string.Equals(i.text, "yes", StringComparison.OrdinalIgnoreCase));
                                 var imagesWithoutText = allImages.Where(i => string.Equals(i.text, "no", StringComparison.OrdinalIgnoreCase));
 
-                                const double desiredAspect = 0.666666667;
+                                const double DesiredAspect = 0.666666667;
```

Comment:
can we do it like this for clarity?

```suggestion
                                const double DesiredAspect = 2./3.;
```